server/chat-plugins/helptickets.ts

```diff
@@ -2992,6 +2992,16 @@ export const commands: Chat.ChatCommands = {
 			`/helpticket public [user], [date] - Makes the ticket logs for the [user] on the [date] public to staff. Requires: &`,
 		],
 	},
+
+	tb: 'ticketban',
+	ticketban(target, room, user) {
+		return this.parse(`/helpticket ban ${target}`);
+	},
+
+	unticketban(target, room, user) {
+		return this.parse(`/helpticket unban ${target}`);
+	},
```

Comment:
```suggestion
	tb: 'ticketban',
	unticketban: 'ticketban',
	ticketban(target, room, user, connection, cmd) {
		return this.parse(`/helpticket ${cmd} ${target}`);
	},
```
(and alias /ht unban to /ht unticketban - this should be a little tidier overall)